services/database/schema-updates/post-v0-to-v1-upgrade.mjs

```diff
@@ -0,0 +1,95 @@
+// This script is used to finish the upgrade of the database schema from v0 to v1
+// The sources table is filled but the source_hash_keccak column is empty
+// This script fills the source_hash_keccak column with the keccak256 hash of the source
+
+import pg from "pg";
+import { keccak256 } from "ethers";
```

Comment:
I think `ethers` should be defined in the package.json of the `database` folder if used here.